src/textarea/textarea.tsx

```diff
@@ -9,25 +10,120 @@ interface Props {
     className?: string;
     placeholder?: string;
     disabled?: boolean;
+    enableCharacterLimit?: boolean;
```

Comment:
Shouldn't that allow an integer? I just saw that the limit is hard coded to 256.

Reply:
fixed